src/pages/workspaces/List.js

```diff
@@ -129,12 +129,14 @@ const NewWorkspaceCard = pure(({ listView, onClick }) => {
 })
 
 
-export const WorkspaceList = ajaxCaller(class WorkspaceList extends Component {
+const listViewAtom = Utils.atom(false)
```

Comment:
Good approach. I think we'll probably also want this for other pages that have card views, so we'll need to decide whether to have separate atoms or one aggregate one with keys.